casper/src/test/scala/coop/rchain/casper/api/BlockQueryResponseAPITest.scala

```diff
@@ -29,192 +27,176 @@ class BlockQueryResponseAPITest
-                randomDeploys.map(_.toDeployInfo)
-              )
-              val b = blockInfo.blockInfo
-              b.blockHash should be(secondBlock.blockHash.toHexString)
-              b.sender should be(secondBlock.sender.toHexString)
-              b.blockSize should be(secondBlock.toProto.serializedSize.toString)
-              b.seqNum should be(secondBlock.toProto.seqNum)
-              b.sig should be(secondBlock.sig.toHexString)
-              b.sigAlgorithm should be(secondBlock.sigAlgorithm)
-              b.shardId should be(secondBlock.toProto.shardId)
-              b.version should be(secondBlock.version)
-              b.blockNumber should be(secondBlock.blockNumber)
-              b.preStateHash should be(
-                secondBlock.preStateHash.toHexString
-              )
-              b.postStateHash should be(
-                secondBlock.postStateHash.toHexString
-              )
-              b.bonds should be(secondBlock.bonds.map(BlockApi.bondToBondInfo))
-              b.blockSize should be(secondBlock.toProto.serializedSize.toString)
-              b.deployCount should be(secondBlock.state.deploys.length)
-              b.justifications should be(secondBlock.justifications.map(_.toHexString))
-          }
-        } yield ()
+  "getBlock" should "return successful block info response" in {
+    implicit val (blockStore, blockDagStorage, runtimeManager) = Mocks.create[Task]()
+
+    for {
+      _                  <- prepareDagStorage[Task]
+      blockApi           <- createBlockApi[Task]("", 1)
+      hash               = secondBlock.blockHash.toHexString
+      blockQueryResponse <- blockApi.getBlock(hash)
+      _ = inside(blockQueryResponse) {
+        case Right(blockInfo) =>
+          blockInfo.deploys should be(
+            randomDeploys.map(_.toDeployInfo)
+          )
+          val b = blockInfo.blockInfo
+          b.blockHash should be(secondBlock.blockHash.toHexString)
+          b.sender should be(secondBlock.sender.toHexString)
+          b.blockSize should be(secondBlock.toProto.serializedSize.toString)
+          b.seqNum should be(secondBlock.toProto.seqNum)
+          b.sig should be(secondBlock.sig.toHexString)
+          b.sigAlgorithm should be(secondBlock.sigAlgorithm)
+          b.shardId should be(secondBlock.toProto.shardId)
+          b.version should be(secondBlock.version)
+          b.blockNumber should be(secondBlock.blockNumber)
+          b.preStateHash should be(
+            secondBlock.preStateHash.toHexString
+          )
+          b.postStateHash should be(
+            secondBlock.postStateHash.toHexString
+          )
+          b.bonds should be(secondBlock.bonds.map(BlockApi.bondToBondInfo))
+          b.blockSize should be(secondBlock.toProto.serializedSize.toString)
+          b.deployCount should be(secondBlock.state.deploys.length)
+          b.justifications should be(secondBlock.justifications.map(_.toHexString))
```

Comment:
Can we reuse existing methods in block API to convert block to LightBlockInfo and just check their instances instead of each field?